woocommerce/woocommerce-ios
# Add an entry point to Inbox in Hub menu

Closes: #6106 


### Description


This PR added a Hub menu item to access Inbox notes based on design vbzgTKHV9XsyyN5cxxAuhv-fi-2496%3A92584 if the Inbox feature flag is enabled with two test cases.

### Testing instructions


- Launch the app
- Go to the Menu tab --> there should be a new "Inbox" menu item before "Coupons" and after "View Store"

---
- [x] @jaclync tests the case when the Inbox feature flag is disabled

### Screenshots


light | dark | landscape
-- | -- | --
![Simulator Screen Shot - iPhone 11 - 2022-02-08 at 15 51 04](https://user-images.githubusercontent.com/1945542/152944080-80db9956-705f-4210-814c-13786a1ab555.png) | ![Simulator Screen Shot - iPhone 11 - 2022-02-08 at 15 51 16](https://user-images.githubusercontent.com/1945542/152944097-68edfee8-0945-4ed4-835a-f40e1481acab.png) | ![Simulator Screen Shot - iPhone 11 - 2022-02-08 at 15 51 21](https://user-images.githubusercontent.com/1945542/152944103-45540952-19b8-4524-b66f-bfd99e69f6b4.png)


---
